idaes/models/properties/examples/tests/test_methanol_ideal_VLE.py

```diff
@@ -0,0 +1,298 @@
+                }
+
+        assert model.params.config.phases == {
+            'Liq': {"type": LiquidPhase,
+                    "equation_of_state": Ideal},
+            'Vap': {"type": VaporPhase,
+                    "equation_of_state": Ideal}
+            }
+
+        assert model.params.config.state_definition == FPhx
+
+        assertStructuredAlmostEqual(
+            model.params.config.state_bounds,
+            {
+                "flow_mol": (1e-10, 100, 1e10, pyunits.mol/pyunits.s),
+                "enth_mol": (-1e10, 100, 1e10, pyunits.J/pyunits.mol),
+                "temperature": (198.15, 298.15, 512.75, pyunits.K),
+                "pressure": (1e-10, 1e5, 1e10, pyunits.Pa),
+            },
+            item_callback=_as_quantity,
+        )
+
+        assert model.params.config.phase_equilibrium_state == {
+            ("Vap", "Liq"): SmoothVLE
+        }
+
+        assert model.params.config.bubble_dew_method == IdealBubbleDew
+
+        assert isinstance(model.params.phase_equilibrium_idx, Set)
+        assert len(model.params.phase_equilibrium_idx) == 1
+        for i in model.params.phase_equilibrium_idx:
+            assert i in [
+                "PE1",
+            ]
+
+        assert model.params.phase_equilibrium_list == {
+            "PE1": {"CH3OH": ("Vap", "Liq")},
+        }
+
+        assert model.params.pressure_ref.value == 1E5
+        assert model.params.temperature_ref.value == 298.15
+
+        assert model.params.CH4.mw.value == 16.043E-3
+        assert model.params.CH4.pressure_crit.value == 46.0e5
+        assert model.params.CH4.temperature_crit.value == 190.4
+
+        assert model.params.CO.mw.value == 28.010E-3
+        assert model.params.CO.pressure_crit.value == 35.0e5
+        assert model.params.CO.temperature_crit.value == 132.9
+
+        assert model.params.H2.mw.value == 2.016E-3
+        assert model.params.H2.pressure_crit.value == 12.9e5
+        assert model.params.H2.temperature_crit.value == 33.0
+
+        assert model.params.CH3OH.mw.value == 32.042E-3
+        assert model.params.CH3OH.pressure_crit.value == 80.9e5
+        assert model.params.CH3OH.temperature_crit.value == 512.6
+        assert_units_consistent(model)
+
+
+class TestStateBlock(object):
+    @pytest.fixture(scope="class")
+    def model(self):
+        model = ConcreteModel()
+        model.params = GenericParameterBlock(default=config_dict)
+
+        model.props = model.params.build_state_block(
+            [1], default={"defined_state": True}
+        )
+
+        # Fix state
+        model.props[1].flow_mol.fix(415.44)
+        model.props[1].enth_mol.fix(-1.3643e5)
+        model.props[1].pressure.fix(5.1e6)
+        model.props[1].mole_frac_comp["CH4"].fix(2.2964e-6)
+        model.props[1].mole_frac_comp["CO"].fix(0.11438)
+        model.props[1].mole_frac_comp["H2"].fix(0.23743)
+        model.props[1].mole_frac_comp["CH3OH"].fix(0.64818)
+
+        assert degrees_of_freedom(model.props[1]) == 0
```

Comment:
I would not put the `assert` in the `fixture`, as this will cause all associated tests to fail. Instead, there should be a separate test for degrees of freedom.